modules/checking-cluster-resource-availability-and-utilisation.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="checking-cluster-resource-availability-and-utilisation_{context}"]
+= Checking cluster resource availability and utilisation
+
+{product-title} provides a comprehensive set of monitoring dashboards that help you understand the state of cluster components.
+
+In the *Administrator* perspective you can access dashboards for core {product-title} components, including:
+
+* etcd
+
+* Kubernetes compute resources
+
+* Kubernetes network resources
+
+* Prometheus
+
+* USE method dashboards relating to cluster and node performance
```

Comment:
What is a _USE_ method?

Reply:
It is a method used when approaching performance issues. It stands for 'utilization, saturation and errors'. My understanding is that the cluster and node performance dashboards provided statistics relating to each of those areas. For the purposes of this bulleted list, I think that a brief summary is suffice. I have subsequently changed that bullet point to the following:

* Dashboards relating to cluster and node performance